agimus_controller/utils/path_finder.py

```diff
@@ -1,7 +1,15 @@
 import rospkg
 from pathlib import Path
+import yaml
 
 
 def get_package_path(package_name) -> Path:
     rospack = rospkg.RosPack()
     return Path(rospack.get_path(package_name))
+
+
+def get_mpc_params_dict():
```

Comment:
in principle you should get these from `agimus_demos_description` as these parameters are specific to 1 demo:
the pick and place one.